compliance/components/ManualProcedures/component.yaml

```diff
@@ -44,32 +44,24 @@ satisfies:
   implementation_status: complete
   narrative:
     - text: |
-        Because the infrastructure is all code, Disaster Recovery into another region
-        should be simple, only requiring a change to the terraform code to change the
-        region and a restore of a database backup from the old region.
+        Because the infrastructure is all code, Disaster Recovery
+        should be simple, only requiring a relaunch and a restore of a database backup
+        from the old system.
 - control_key: PL-8
   standard_key: NIST-800-53
   covered_by: []
   implementation_status: partial
   narrative:
     - text: |
-        The [DevSecOps Architecture](https://github.com/18F/gcp-appengine-template/blob/master/DEVSECOPS.md)
-        describes the overall system and security architecture.  In particular, the
-        [DevSecOps Requirements](https://github.com/18F/gcp-appengine-template/blob/master/DEVSECOPS.md#devsecops-requirements)
-        section goes into details about the DevSecOps security requirements that this project
-        fulfills.
-
-        XXX Be sure to add information about your app's security architecture either here or in
-        the [DevSecOps Architecture](https://github.com/18F/gcp-appengine-template/blob/master/DEVSECOPS.md)
-        document.
+        XXX Be sure to add information about your app's security architecture.
```

Comment:
Describing this from scratch feels like a bit lift for the vendor- aside from the GCP are there any TTS developed SaaS apps with gitflow that we could use to point to this?

Reply:
If anything i think we'd want to reference the workflows.md as the start

Reply:
I don't think that this is a big lift at all.  They own the app, and ought to be able to document all that.  :-)  But I'll put in some references to workflows and so on.